### PR TITLE
Adding support for Instagram embeds

### PR DESCRIPTION
**Pull request by rcrphillips**
## What does this change?
Adds support for the instagram block type. Note that @gtrufitt approved the use of the !important override. In turn, we fix a bug that had existed in frontend.
Screen Shot 2020-04-28 at 14 27 29

Screen Shot 2020-04-28 at 14 25 45

## Why?
For parity

## Link to supporting Trello card
https://trello.com/c/YubOCveT/1547-add-instagram-embeds

### CODE REVIEW

**Comment by rcrphillips** [comment]
[Add instagram embeds](https://trello.com/c/YubOCveT/1547-add-instagram-embeds)

File: src/web/components/elements/InstagramBlockComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { unescapeData } from '@root/src/lib/escapeData';
+import { css } from 'emotion';
+
+const instagramoverride = css`
```
Comment (oliverlloyd) [inline src/web/components/elements/InstagramBlockComponent.tsx:6]: Maybe `overrideWidth` to be more descriptive?

Comment (rcrphillips) [reply src/web/components/elements/InstagramBlockComponent.tsx]: Good shout.

**Comment by rcrphillips** [comment]
@oliverlloyd I'm not really au fait with the testing side of things on dcr, but your suggestion sounds sensible...... Would that be ok in a separate PR?

**Comment by oliverlloyd** [comment]
@rcrphillips I added a quick test to check for an instagram embed but I'm not seeing them it appear when I run this locally (so the test is failing). Am I using the right kind of test article?

https://www.theguardian.com/lifeandstyle/2020/apr/28/caroline-calloway-survived-being-cancelled

Looking at the blocks for this article the instagram embeds here are coming through as `EmbedBlockElement`s, which aren't supported in `ArticleRender` so maybe sometimes they get set as this and sometimes as `InstagramBlockElement`?

**Comment by gtrufitt** [comment]
> Looking at the blocks for this article the instagram embeds here are coming through as `EmbedBlockElement`s, which aren't supported in `ArticleRender` so maybe sometimes they get set as this and sometimes as `InstagramBlockElement`?

Correct, here’s a couple with InstagramBlockElement

https://www.theguardian.com/media/2018/aug/29/flat-tummy-instagram-women-appetite-suppressant-lollipops


https://www.theguardian.com/lifeandstyle/2019/jul/01/sober-curious-alcohol-abuse

**Comment by oliverlloyd** [comment]
> here’s a couple with InstagramBlockElement

Thanks, that did it. Test passed now 👍

**Pull request merged**